### ISSUE TITLE
Raspberry Pi

### ISSUE THREAD

**Issue by rockcat** [tebl/RC6502-Apple-1-Replica#4]
Hi

You might like this: https://github.com/rockcat/RPi_RC6502

Cheers

**Comment by tebl** [comment]
That is pretty cool, thanks! Wonder if it'd work with a Pi Zero and a small adapter board, would look nice to have it slotted on there :-)

**Comment by rockcat** [comment]
Yes should work with any Pi, you could remote in over wifi too :)
